tslearn/docs/examples/plot_dtw.py

```diff
@@ -5,6 +5,14 @@
 
 This example illustrates DTW computation between time series and plots the
 optimal alignment path.
+
+The image represents cost matrix, that is the squared Euclidean distance for
+each time point between both time series, which are represented
+at the left and at the top of the cost matrix.
+
+The optimal path, that is the path that minimizes the total cost to go from
+the first time point to the last one, is represented in white on the image.
+
```

Comment:
Maybe a good thing would be to add scholar references to the examples, WDYT?

Reply:
Similarly, maybe adding a ref to the LB_Keogh doc would help.

Reply:
I will change that!